tracking/src/main/java/org/motechproject/nms/tracking/cache/CacheHelper.java

```diff
@@ -0,0 +1,5 @@
+package org.motechproject.nms.tracking.cache;
+
+public interface CacheHelper {
```

Comment:
This class should be called EntityServiceWrapper since that is what it is.  Spring provides the cache help for us.